.buildkite/pipeline-release-operatorhub.yml

```
@@ -31,20 +15,29 @@ steps:
         key: "redhat-container-push"
         depends_on:
           - "generate-crds"
+          - "build-operatorhub-tool"
         commands:
-          - buildkite-agent artifact download "bin/operator*" /usr/local/
-          - /usr/local/bin/operatorhub container push --dry-run=false
+          - buildkite-agent artifact download "bin/operatorhub" /usr/local/
+          - chmod u+x /usr/local/bin/operatorhub
+          - cd hack/operatorhub
+          - operatorhub container push --dry-run=\${DRY_RUN:-true}
```

```suggestion
          - operatorhub container push --dry-run=$${DRY_RUN:-true}
```

Are you sure this works?  IIRC you have to double the `$` in this yml file.

---

We can escape the $ character using `$$` or `\$` (https://buildkite.com/docs/pipelines/environment-variables#runtime-variable-interpolation).

Live test: https://buildkite.com/elastic/cloud-on-k8s-operator/builds/2208#0186dc76-80b9-44f6-bf95-53804bc91d13

```chmod u+x /usr/local/bin/operatorhub
--
  | cd hack/operatorhub
  | operatorhub container push --dry-run=${DRY_RUN:-true}
  | 2023-03-13 19:35:57 INFO   Searching for artifacts: "bin/operatorhub"
  | 2023-03-13 19:35:58 INFO   Found 1 artifacts. Starting to download to: /usr/local
  | 2023-03-13 19:35:59 INFO   Successfully downloaded "bin/operatorhub" 65039210 bytes
  | 2023/03/13 19:36:00 Determining if image already exists within project with tag: 2.7.0-SNAPSHOT
  | 2023/03/13 19:36:01 not pushing image as dry-run is set.
  | 2023/03/13 19:36:01 ✓
```

---

interesting... I swear this documentation, or this feature just changed, as I ran into this when dealing with the operatorhub changes. regardless TIL.